theme/style.css

```diff
@@ -16,3 +16,7 @@
 }
 
 /* style your grid sections here!  */
+p {
```

Comment:
`p` is too broad, please make this selector more specific (e.g. with an ID or an `nth-child` selector)

Reply:
hey i fixed it ;-)

Reply:
finalized for sure ;-)